docs/user/alerting/action-types/slack.asciidoc

```diff
@@ -26,21 +26,15 @@ Webhook URL::   The URL of the incoming webhook. See https://api.slack.com/messa
      webhookUrl: 'https://hooks.slack.com/services/abcd/efgh/ijklmnopqrstuvwxyz'
 --
 
-**`secrets`** defines sensitive information for the action type and contains the following properties:
+Secrets defines sensitive information for the action type.
 
-[cols="2*<"]
-|===
-
-| `webhookUrl`
-| A string that corresponds to *Webhook URL*.
-
-|===
+`webhookUrl`:: A string that corresponds to *Webhook URL*.
 
 [float]
 [[slack-action-configuration]]
 ==== Action configuration
 
-Slack actions have the following properties:
```

Comment:
Configure a Slack account

This sentence needs to be fixed: 

You configure the accounts Slack action type can use to communicate with Slack in the connector form.

The image under step 5 is too small.